modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/utils/odata.es.js

```diff
@@ -312,6 +400,42 @@ function transformGroupNode(context) {
 		}];
 }
 
+/**
+ * Transform an operator expression node into a criterion for the criteria
+ * builder.
+ * @param {object} oDataASTNode
+ * @returns an array containing the object representation of an operator
+ * criterion
+ */
+function transformNotNode({oDataASTNode}) {
+	const nextNodeExpression = getNextOperatorExpression(oDataASTNode);
+
+	const nextNodeExpressionName = getExpressionName(nextNodeExpression);
+
+	let returnValue;
+
+	if (nextNodeExpressionName == OPERATORS.CONTAINS) {
+		returnValue = [
+			{
+				operatorName: NOT_OPERATORS.NOT_CONTAINS,
+				propertyName: nextNodeExpression.value.parameters[0].raw,
+				value: nextNodeExpression.value.parameters[1].raw.replace(/['"]+/g, '')
```

Comment:
Since this regex is used quite a bit, lets just set it as a constant in this file. And possibly even create a helper function for removing quotes.

Reply:
I'll create a helper function for this. It'll be more explicit that way as well.